imports/lib/composeWithTracker.js

```diff
@@ -0,0 +1,32 @@
+//
+// compose-with-tracker.js
+//
+import {compose} from "react-komposer";
```

Comment:
React Komposer hasn't been updated in over a year. Why not use recompose's [`compose`](https://github.com/acdlite/recompose/blob/master/docs/API.md#compose)?

Reply:
I have just used a drop in replacement to slowly make it compatible with react v16.

Reply:
I think it's better to remove react-komposer entirely and use only withTracker from react-meteor-data.
It's simple and efficient.